cmd/catalog.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/ocdev/pkg/catalog"
+	"github.com/spf13/cobra"
+)
+
+var catalogCmd = &cobra.Command{
+	Use:   "catalog [options]",
+	Short: "Catalog related operations",
+}
+
+var catalogListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List all available component types.",
+	Long:  "List all available component types.",
+	Example: `
+# Get the supported components
+ocdev catalog list
+
+# Deploy the component
```

Comment:
There should be only examples for  this command

Reply:
done